apps/ejabberd/include/mod_muc_room.hrl

```diff
@@ -88,7 +88,15 @@
                 room_queue = queue:new()
                }).
 
--record(muc_online_users, {us :: ejabberd:simple_bare_jid(),
-                           room :: mod_muc:room(),
-                           host :: ejabberd:server()
-                          }).
+-record(muc_online_users, {
+          us,
+          room,
+          host
+         }).
+
+-type muc_online_users() :: #muc_online_users{
+        us :: ejabberd:simple_bare_jid(),
+        room :: mod_muc:room(),
```

Comment:
Like in case of session record - precise specs directly in the record make Dialyzer unhappy about e.g. `dirty_select`